js/main.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-use-before-define */
+// Create array of books
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const bookAdd = document.querySelector('#available-books');
+const btn = document.querySelector('#add-button');
+
+if (!localStorage.getItem('bookInfo')) {
+  localStorage.setItem('bookInfo', JSON.stringify([]));
+}
+
+let books;
+
+function saveBooks(book) {
+  localStorage.setItem('bookInfo', JSON.stringify(book));
+}
+
+/* eslint-disable no-use-before-define */
+
+function displayBookData() {
+  books = JSON.parse(localStorage.getItem('bookInfo'));
+  userUpdate();
+}
+displayBookData();
+
+function removeBook() {
+  books.splice(this, 1);
```

Comment:
- [OPTIONAL] Use array methods that doesn't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice). In this case the `filter `method should be the good to go solution

Reply:
Hi, @EricMbouwe
Thanks. Well noted.